### PR TITLE
AO-744: Extend Admin Products and Facilities pages with print option

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://openlmis.atlassian.net/browse/AO-744

### CODE REVIEW

File: src/admin-facility-list/messages_en.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "adminFacilityList.code": "Code",
+    "adminFacilityList.type": "Type",
+    "adminFacilityList.active": "Active",
+    "adminFacilityList.enabled": "Enabled",
+    "adminFacilityList.actions": "Actions",
+    "adminFacilityList.search": "Search",
+    "adminFacilityList.addFacility": "Add Facility",
+    "adminFacilityList.edit": "Edit",
+    "adminFacilityList.print": "Print"
+}
```
Comment (dariapionk) [inline src/admin-facility-list/messages_en.json:15]: There is no need to copy the whole messages_en.json file, you can just add  "adminFacilityList.print": "Print"

File: src/admin-orderable-list/messages_en.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "adminOrderableList.name": "Name",
+    "adminOrderableList.description": "Description",
+    "adminOrderableList.program": "Program",
+    "adminOrderableList.search": "Search",
+    "adminOrderableList.products.error": "No products found",
+    "adminOrderableList.actions": "Actions",
+    "adminOrderableList.edit": "Edit",
+    "adminOrderableList.addProduct": "Add Product",
+    "adminOrderableList.print": "Print"
+}
```
Comment (dariapionk) [inline src/admin-orderable-list/messages_en.json:13]: There is no need to copy the whole messages_en.json file, you can just add"adminOrderableList.print": "Print"

File: src/admin-orderable-print/_orderable-print.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.data-download {
+    font-weight: normal;
+    font-size: 14px;
+}
```
Comment (dariapionk) [inline src/admin-orderable-print/_orderable-print.scss:4]: I am not sure, but I think that one scss file is enough as the scope for scss files is whole application. Could you please check?

Comment (sniedzielski) [reply src/admin-orderable-print/_orderable-print.scss:4]: I think that we can keep like this and in the future try to move this into global scss file.

File: src/admin-orderable-list/messages_en.json (hunk 2 above)
Comment (sniedzielski) [reply src/admin-orderable-list/messages_en.json:13]: ok, I will change this

File: src/admin-facility-list/messages_en.json (hunk 1 above)
Comment (sniedzielski) [reply src/admin-facility-list/messages_en.json:15]: ok, I will change this

**Review by dariapionk — APPROVED** [review]

**Comment by sniedzielski** [comment]
@dariapionk can you re-check?

**Pull request merged**